package.json

```diff
@@ -108,7 +108,7 @@
     "node-html-encoder": "0.0.2",
     "remap-istanbul": "0.6.4",
     "request": "2.75.0",
-    "resolve-bin": "0.4.0",
+    "resolve-bin": "^0.4.0",
```

Comment:
Can you lock this version to the latest one that works for you? We want to avoid running into issues with versions automatically updating.